src/reference/02-DetailTopics/01-Using-sbt/01-Command-Line-Reference.md

```diff
@@ -12,9 +13,8 @@ out: Command-Line-Reference.html
   [Commands]: Commands.html
   [Running-Project-Code]: Running-Project-Code.html
   [Testing]: Testing.html
-  [Sbt-Launcher]: Sbt-Launcher.html
 
-Command Line Reference
+Sbt prompt command Line Reference
```

Comment:
This page I think is sort of written in the style of a manpage for sbt, so I think Command Line Reference sort of makes sense as the title.